src/components/modal-product-limits/ModalProductLimits.tsx

```diff
@@ -1,30 +1,25 @@
-import { createPortal } from 'react-dom'
 import styles from './ModalProductLimits.module.scss'
 import IconArrowClose from '../../icons/closemodal.svg?react'
 import IconTriangle from '../../icons/triangle.svg?react'
 import { FC } from 'react'
 
-const modalRoot = document.querySelector(
-  '#modal-root-product-limits'
-) as HTMLElement
-
 type Props = {
   onClick: () => void
+  customStyles: string
```

Comment:
it should be `optional` and styles means that it would go to `styles={}` prop, not to `className`

Reply:
done